plugins/woocommerce/includes/admin/settings/views/html-admin-page-shipping-zone-methods.php

```diff
@@ -148,7 +148,7 @@
 				</article>
 				<footer>
 					<div class="inner">
-						<button id="btn-ok" data-status='{{ data.status }}' class="button button-primary button-large">
+						<button id="btn-ok" data-status='{{ data.status }}' disable='{{ data.status === "new" ? "disabled" : "" }}' class="button button-primary button-large {{ data.status === 'new' ? 'disabled' : '' }}">
```

Comment:
Typo perhaps, it seems like you can still click on `Create` despite it looks like disabled.

```suggestion
						<button id="btn-ok" data-status='{{ data.status }}' disabled='{{ data.status === "new" ? "disabled" : "" }}' class="button button-primary button-large {{ data.status === 'new' ? 'disabled' : '' }}">
```

Reply:
Whoops, nice catch